### DISCUSSION TITLE
Survival Tree

### DISCUSSION THREAD

**Discussion by danieleacampos** [sebp/scikit-survival#210] [Q&A]
I am trying to do an analysis with a survival database for my doctor degree. I would like to move for machine learnig approach. In Brazil, they are not comum for survival databases. I would like to start with a decision tree and than a random florest, because i have read "A review of survival trees" by Imad Bou-Hamad and and "RATIONALE AND APPLICATIONS OF SURVIVAL TREE AND 
SURVIVAL ENSEMBLE METHODS" by Yan Zhou. 
I first used sksurv.tree.SurvivalTree and it works.  However i would like to see the tree. I tried use the sklearn tree like refered in The "underlying Tree object. Please refer to help(sklearn.tree._tree.Tree) for attributes of Tree object.". But i couldn't print it. Is it possible?

**Comment by sebp** [comment]
Have a look at https://github.com/sebp/scikit-survival/issues/110
